### ISSUE TITLE
Managed node groups missing outputs

### ISSUE THREAD

**Issue by TBeijen** [terraform-aws-modules/terraform-aws-eks#643]
# I have issues

## I'm submitting a...

* [ ] bug report
* [x] feature request
* [ ] support request - read the [FAQ](https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/docs/faq.md) first!
* [ ] kudos, thank you, warm fuzzy

## What is the current behavior?

Currently only iam roles added to the managed nodes is returned.
[Terraform module outputs various other outputs](https://www.terraform.io/docs/providers/aws/r/eks_node_group.html#attributes-reference) including resulting asg arn, status, etc.

Especially the asg arn seems useful.

## What's the expected behavior?

Output all output returned by Terraform module, as well as relevant values set by the terraform-aws-eks module (such as the resulting iam role)

## Are you able to fix this problem and submit a PR? Link here if you have already.

Will add PR shortly (WIP first, stating what the output looks like)

**Comment by TBeijen** [comment]
First local output example:
```
# work in progress:
eks_cluster_apps1_node_groups = [
  {
    "arn" = "arn:aws:eks:eu-west-1:NNNNNNNN:nodegroup/apps-1/apps-1-ng-managed-a-rapid-skylark/nnnnnnnn-nnnn-nnnn-nnnn-nnnnnnnnnnnn"
    "id" = "arn:aws:eks:eu-west-1:NNNNNNNN:nodegroup/apps-1/apps-1-ng-managed-a-rapid-skylark/nnnnnnnn-nnnn-nnnn-nnnn-nnnnnnnnnnnn"
    "resources" = [
      {
        "autoscaling_groups" = [
          {
            "name" = "eks-nnnnnnnn-nnnn-nnnn-nnnn-nnnnnnnnnnnn"
          },
        ]
        "remote_access_security_group_id" = "sg-nnnnnnnnnnn"
      },
    ]
    "status" = "ACTIVE"
  },
]


# Added by #621:
eks_cluster_apps1_node_groups_iam_role_arns = {
  "apps-1-ng-managed-a-rapid-skylark" = "arn:aws:iam::NNNNNNNN:role/apps-1-managed-node-groups"
}
```

The map returning the role arn, as added via #641, has as key a computed value that can't be predicted. I would suggest having the `node_group` outputs either:

* Ordered list with maps, same order as node_groups are defined.
* Map where key is the `name` variable used in node group definition.

**Comment by dpiddockcmp** [comment]
This was fixed in #650 and released in v8.0.0
